aws-s3-bucket.yml

```diff
@@ -20,26 +20,26 @@ display_name: AWS S3 Bucket
 image_url: https://www.pikpng.com/pngl/m/73-731373_aws-s3-logo-png-transparent-amazon-s3-logo.png
 documentation_url: https://aws.amazon.com/s3/
 support_url: https://aws.amazon.com/s3/
-tags: [aws, s3, preview]
+tags: [ aws, s3, preview ]
 plan_updateable: true
 plans:
-- name: private
-  id: 8938b4c0-d67f-4c34-9f68-a66deef99b4e
-  description: 'Private S3 bucket'
-  display_name: 'Private'
-  properties:
-    acl: private
-- name: public-read
-  id: 04317eaa-11ac-4c5f-b77f-eb005fe977fe
-  description: 'Public-read S3 bucket'
-  display_name: 'Public Read'
-  properties:
-    acl: public-read
+  - name: private
```

Comment:
There seem to be a lot of formatting changes that make this yaml file not consistent with the others. If you used goland I think the go formatted makes this change on save - if that's the case you can exclude .yml files from the formatting rules. I would suggest you revert the formatting to what it was to keep it consistent with other files and to keep the changes in this PR to a min.

Reply:
Done!! Thank you very much!